go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/cortexproject/cortex
 
-go 1.22
+go 1.22.0
+
+toolchain go1.22.5
```

Comment:
Is there a way to avoid this? If not, then we can live it...

Reply:
I'm not familiar with this.
Thanos also had the same:
- https://github.com/thanos-io/promql-engine/blob/main/go.mod#L5
- https://github.com/thanos-io/thanos/blob/main/go.mod#L5

Reply:
Related https://github.com/golang/go/issues/65847

Reply:
Let's live with it then. I see Prometheus has the same config.